firmware/ubirch/m66/m66_http.c

```diff
@@ -24,9 +24,13 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <ubirch/timer.h>
+#include <stdio.h>
 #include "ubirch/modem.h"
 #include "m66_debug.h"
 
+
+char file_name[] = "RAM:file.txt";
```

Comment:
change this to something like file.bin as we are probably not downloading any text files, keeps confusion low.